hedera-mirror-rest/monitoring/monitor_apis/utils.js

```diff
@@ -106,9 +116,10 @@ const fetchWithRetry = async (url, opts = {}) => {
  * Host info is prepended to if only path is provided
  * @param {*} url rest-api endpoint
  * @param {String} key JSON key of the object to return
+ * @param {Function} retryPredicate An optional predicate that returns true if a successful request should be retried.
  * @return {Object} JSON object representing api response or error
  */
-const getAPIResponse = async (url, key = undefined) => {
+const getAPIResponse = async (url, key = undefined, retryPredicate = undefined) => {
```

Comment:
can add a default noRetry predicate `const noRetry = () => false;`, then set as the default value here

```suggestion
const getAPIResponse = async (url, key = undefined, retryPredicate = noRetry) => {
```

then in `fetchWithRetry` can change to

```
      if (!retryPredicate(json)) {
        return json;
      }
```

Reply:
Done